### ISSUE TITLE
Some warning when training

### ISSUE THREAD

**Issue by lubin202209** [DerrickXuNu/OpenCOOD#72]
Hello, when I try to train the model, the terminal printed out the warning that is "Userwarning: Detected call of 'lr_scheduler.step()' before 'optimizer.step()'. In PyTorch 1.1.0 and later, you should call them in the opposite order: 'optimizer.step()' before 'lr_scheduler.step()'. Failure to do this will result in PyTorch skipping the first value of the learning rate schedule." and "Userwarning: The epoch parameter in 'scheduler.step()' was not necessary and is being deprecated where possible. Please use 'scheduler.step()' to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainble form, where available. " Do you know what's the problem with these? Is the version of pytorch I installed wrong? The version I installed are pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 
![mmexport1681351005861](https://user-images.githubusercontent.com/116709464/231629342-26c6f0b4-8ebd-4a26-8a05-92f44aaeaad9.png)
Besides these, when I try to train the point_pillar_fcooper model, the terminal printed out three lines of "too many cavs" after it printed out "--------------Dataset Building--------------", Will this have any impact on the entire training process and results?
![mmexport1681351874951](https://user-images.githubusercontent.com/116709464/231629465-5fbc4fb0-64a6-4f0c-8bed-961ae04e4814.png)

**Comment by DerrickXuNu** [comment]
Hi, you could ignore those warnings as they have no effects in the training
